producer.py

```diff
@@ -181,6 +185,17 @@ def package_releases_update(
                     package_index.url,
                     only_if_package_seen=only_if_package_seen,
                 )
+                async_tasks.append(
+                    package_released.publish_to_topic(
+                        package_released.MessageContents(
+                            package_name=package_name,
+                            package_version=package_version,
+                            index_url=[package_index.url],
+                            component_name="package_releases_job",
```

Comment:
@KPostOffice do we want to namespace component name in these messages? e.g. `thoth.package_releases_job`?

Reply:
thanks, https://github.com/thoth-station/package-releases-job/pull/449/commits/9fbb260d611963e8f21c2809f45950056fb482bb